sonarqube-scan/action.yml

```diff
@@ -30,15 +51,67 @@ runs:
       uses: actions/download-artifact@v3
       with:
         name: "${{ github.sha }}-lint-results.out"
-    - name: Get GitHub short SHA as version
-      id: get-short-sha
-      run: echo "SHORT_SHA=${GITHUB_SHA::7}" >> $GITHUB_OUTPUT
-      shell: bash
     - name: SonarQube Scan
       uses: sonarsource/sonarqube-scan-action@master
       env:
         SONAR_TOKEN: ${{ inputs.SONAR_TOKEN }}
         SONAR_HOST_URL: ${{ inputs.SONAR_HOST_URL }}
       with:
         args: >
-          -Dsonar.projectVersion=${{ steps.get-short-sha.outputs.SHORT_SHA }}
+          -Dsonar.projectVersion=${{ steps.get-short-sha.outputs.short_sha }}
+    - name: Notify slack channel on failure
+      if: failure() && inputs.SLACK_CHANNEL_ID != null
+      # if: failure() && github.ref == 'refs/heads/main'
```

Comment:
Do we need this commented out code?

Reply:
Whoops, no. I meant to ask if we want this stuff still just for main if it's on a dev channel, but I'll put it back how it was for now.